app/models/set_member_subject.rb

```diff
@@ -4,7 +4,6 @@ class SetMemberSubject < ActiveRecord::Base
   
   belongs_to :subject_set, counter_cache: true, touch: true
   belongs_to :subject
-  has_many :classifications
```

Comment:
Also processsing from Kafka saves us a bunch of INSERTs on each classification we save.